ptclient/.eslintrc.js

```diff
@@ -18,11 +18,11 @@ module.exports = {
   // add custom rules here
   rules: {
     /*
-      Q) Why needed? 
+      Q) Why needed?
       - before every build eslint loader check all files and return format errors and warnings (eg:  no-console, whitspace ...) in console.
       - We do not want to show these errors in production environment as private data may leak.
     */
-    'no-console': process.env.NODE_ENV === 'production' ? 'off' : 'warning', // stop showing console warning from production enviroment
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'off' : 'warning', // stop showing debugger warning from production enviroment
+    'no-console': process.env.NODE_ENV === 'production' ? 'off' : 'warn', // stop showing console warning from production enviroment
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'off' : 'warn', // stop showing debugger warning from production enviroment
```

Comment:
`'warning'` is not a valid console method. This should be `'warn'`, which will now throw the correct warnings during runtime.